File: apps/operations/lib/core/models/expense_category.dart
abstract final class ExpenseCategory {
  static bool isSalary(Object? value) => const {
    'პერსონალი',
    'ხელფასი',
    'ხელფასები',
    'salary',
    'salaries',
  }.contains(value?.toString().trim().toLowerCase());
  static bool isProcurement(Object? value) => const {
    'market',
    'ბაზარი',
    'მარკეტი',
    'შესყიდვა',
    'შესყიდვები',
  }.contains(value?.toString().trim().toLowerCase());
}

File: apps/backend/src/mobile/util/expense-category.ts
/** Legacy procurement categories have no independent Expense meaning. */
export const isProcurementCategory = (value: unknown): boolean =>
  ['market', 'ბაზარი', 'მარკეტი', 'შესყიდვა', 'შესყიდვები'].includes(
    String(value ?? '')
      .trim()
      .toLowerCase(),
  );
export const isSalaryCategory = (value: unknown): boolean =>
  ['პერსონალი', 'ხელფასი', 'ხელფასები', 'salary', 'salaries'].includes(
    String(value ?? '')
      .trim()
      .toLowerCase(),
  );
